api/go.mod

```diff
@@ -1,13 +1,13 @@
 module github.com/apisix/manager-api
 
-go 1.13
+go 1.16
```

Comment:
IMHO We should not change the lowest GO version, do you use any new features in Go/1.16?

Reply:
This is an unexpected change. Maybe due to some functions of IDE, I will change it back.